README.md

```diff
@@ -14,6 +14,21 @@ You can find us on Twitter [@MozillaReality](https://twitter.com/mozillareality)
 git clone git@github.com:MozillaReality/FirefoxReality.git
 ```
 
+*Clone the third-party repo.*
+
+If you're developing for the Oculus, Snapdragon VR, or VIVE, you need to clone the repo with third-party SDK files. 
+
+```bash
+git clone git@github.com:MozillaReality/FirefoxReality-android-third-party.git third_party
+```
+
+This repo is only available to Mozilla employees. If you have access to the relevant SDK but not this repo, you can manually place them here:
+
+ - `third_party/ovr_mobile/` for Oculus (should contain a `VrApi` folder)
```

Comment:
This actually won't work unfortunately. The SDK places the libraries in `ovr_mobile_sdk_1.17.0/VrApi/Libs/Android/armeabi-v7a/Release/` but we look for them in `third_party/ovr_mobile/VrApi/Libs/Android/armeabi-v7a/`. This is something I wanted to fix in our private repo but haven't had time.